src/flib/pio_nf.F90

```diff
@@ -1219,16 +1219,17 @@ integer function inq_varname_id(ncid                      ,varid,name) result(ie
     integer                                                 , intent(in) :: varid
     character(len=*)                                        , intent(out)    :: name
     interface
-       integer(C_INT) function PIOc_inq_varname(ncid        ,varid,name) &
+       integer(C_INT) function PIOc_inq_varname(ncid        ,varid,name,namelen) &
```

Comment:
We can't actually change the C API without breaking our similarities to the netCDF C API. Those similarities are going to be exploited when I package PIO as a dispatch layer of netCDF, so we can't break them.